lib/libsys/closefrom.2

```diff
@@ -60,6 +60,10 @@ Supported
 .It Dv CLOSE_RANGE_CLOEXEC
 Set the close-on-exec flag on descriptors in the range instead of closing them.
 .El
+.Bl -tag -width ".Dv CLOSE_RANGE_CLOFORK"
```

Comment:
Why do you start new list instead of adding an item to existing one?
I.e., remove the .El above and .Bl lines.